src/abstractdataframe/show.jl

```diff
@@ -581,15 +581,23 @@ end
 showcols(df::AbstractDataFrame) = showcols(STDOUT, df) # -> Void
 
 using Juno
-using Juno: Inline, LazyTree, Table, Row, strong
+using Juno: Inline, LazyTree, Table, Row, strong, undefs
 
 const SIZE = 25
 
+function tomat(df::AbstractDataFrame)
```

Comment:
Ah, indeed. Let's keep it that way, then, but please remove `tomat` to something more explicit (`to_matrix`?), as this one reads too much like 🍅.